### DISCUSSION TITLE
Add support for dynamic web pages.

### DISCUSSION THREAD

**Discussion by tur11ng** [wifiphisher/wifiphisher#1579] [Ideas]
I think that ideally there should allow the user to use its custom back end and support custom dynamic web pages. This can probably be implemented by allowing the user to provide a specific port that the victim will be redirected to and then it should be up to the user to handle the rest of the communication.

Going a bit further, this thought can be extended to split the wifiphisher/wifiphisher project to two modules/projects, one covering the attack and one covering the phishing procedures.

What do you think? If you agree, I can implement the above.

**Comment by Stryker8011** [comment]
Please, think I see where your going. Seems your very capable, so if this is your diagnosis I would like to see the implementation to which your speaking. Thank u for taking a moment, and i have another query if you wouldn't mind.
